openscap_report/scap_results_parser/parsers/rule_parser.py

```diff
@@ -20,10 +50,18 @@ def __init__(self, root, test_results, ref_values):
 
     @staticmethod
     def _get_references(rule):
+        url_to_ref_ids = collections.defaultdict(list)
+        for reference_el in rule.findall(".//xccdf:reference", NAMESPACES):
+            url = reference_el.get("href")
+            if url is None:
```

Comment:
Here is a little catch. I found out that in the `reference` element is present the `href` attribute but with an empty string so it looks like this: `<reference href="">Example</reference>`. So in a variable `url` is a present empty string. Please add a check for the empty string to the if condition.